internal/provider/resource_aci_rest_managed.go

```diff
@@ -289,7 +294,15 @@ func (r *AciRestManagedResource) Read(ctx context.Context, req resource.ReadRequ
 			)
 			return
 		}
-		rn_values = strings.Split(rnMap["rn_values"], ",")
+
+		// Angle brackets are not allowed within ACI class object identifier fields.
+		// Because of that "<,>" string was used to merge and split the list elements.
+		// Only using "," is not enough when the object identifier contains "," for example: "annotationKey-[~!$([])_+-={};:|,.]"
```

Comment:
perhaps explain that this is a limitation of passing a string with private state between import and read?